etc/omero.properties

```diff
@@ -258,7 +258,7 @@ omero.scripts.cache.cron=0 0 0 * * ?
 # for how long.
 #
 # For more information, see
-# http://docs.guava-libraries.googlecode.com/git/javadoc/com/google/common/cache/CacheBuilderSpec.html
+# https://github.com/google/guava/blob/master/guava/src/com/google/common/cache/CacheBuilderSpec.java
```

Comment:
I can use the api doc link instead if you prefer, doesn't seem to allow linking to the actual class though.

I suppose I could try opening an issue and asking what's going on with their javadoc links...